src/App.css

```diff
@@ -1,38 +1,112 @@
-.App {
+@import url('https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,400;0,500;0,600;0,700;0,800;1,500;1,600;1,700&display=swap');
+
+* {
+  font-family: 'Raleway', sans-serif;
+}
+
+body {
+  padding: 1rem;
+  width: 500px;
+  background-color: #202020;
+  color: white;
+}
+
+.searchbar {
+  margin: .5rem 0;
+  padding: .5rem 1rem .5rem 2rem;
+  width: 100%;
+  border-radius: 3px;
+  outline: none;
+  border: 2px solid #a8dadc50;
+  transition: .3s all;
+  background-color: #ffffff25;
+}
+
+.searchbar::placeholder {
+  color: #ffffff75;
+}
+
+.searchbar:focus {
+  border: 2px solid var(--primary-red);
+}
+
+.bat-card {
+  font-size: 14px;
+}
+
+
+/* card img */
+
+
+.bat-card-horizontal.bat-card img {
+  width: 180px !important;
```

Comment:
is important necessary to use here?

Reply:
yeah, due to some reason, but I'll check again.